### DISCUSSION TITLE
Minimum browser versions

### DISCUSSION THREAD

**Discussion by unknownboy** [Authenticator-Extension/Authenticator#767] [Q&A]
Hi,
Where can I get a list of the browser requirements ?

Minimum version for firefox  (57+) can be found in the extension changelog : 
https://addons.mozilla.org/en-US/firefox/addon/auth-helper/versions/

But what is the minimum version for Chrome and Edge ?
Does this extension works on all operating systems (macOS, *nix, win Arm) ?

**Comment by mrx23dot** [comment]
Actually on Firefox it's >v115 that's required.
https://github.com/Authenticator-Extension/Authenticator/discussions/1268
